NEWS.md

```diff
@@ -44,7 +44,7 @@ Language changes
   a function like other operators. The dotted version `.-->` is now parsed as well.
   For backwards compatibility, `-->` still parses using its own expression head
   instead of `:call`.
-* The `a[begin]` syntax now calls `firstindex(a,1)` rather than `first(axes(a,1))` ([#35779]).
+* The `a[begin,k]` syntax now calls `firstindex(a,1)` rather than `first(axes(a,1))` ([#35779]), but the former now defaults to the latter for any `a` ([#38742]).
```

Comment:
```suggestion
* The `a[begin, k]` syntax now calls `firstindex(a, 1)` rather than `first(axes(a, 1))` ([#35779]), but the former now defaults to the latter for any `a` ([#38742]).
```